src/components/events/partials/ModalTabsAndPages/DetailsMetadataTab.tsx

```diff
@@ -50,8 +52,27 @@ const DetailsMetadataTab = ({
 
 	const user = useAppSelector(state => getUserInformation(state));
 
-	const handleSubmit = (values: { [key: string]: any }, catalog: MetadataCatalog) => {
-		dispatch(updateResource({id: resourceId, values, catalog}));
+	const handleSubmit = async (values: { [key: string]: any }, catalog: MetadataCatalog) => {
+		dispatch(updateResource({id: resourceId, values, catalog}))
+			.unwrap()
+			.then(() => {
+				dispatch(addNotification({
+					type: "info",
+					key: "METADATA_SAVED",
+					duration: 3,
+					parameter: undefined,
+					context: NOTIFICATION_CONTEXT,
+				}));
+			})
+			.catch(() => {
+				dispatch(addNotification({
+					type: "warning",
+					key: "METADATA_NOT_SAVED",
+					duration: 3,
+					parameter: undefined,
+					context: NOTIFICATION_CONTEXT,
+				}));
+			})
```

Comment:
Interesting! I would prefer

```suggestion
	const handleSubmit = async (values: { [key: string]: any }, catalog: MetadataCatalog) => {
		try {
			await dispatch(updateResource({ id: resourceId, values, catalog })).unwrap();
			
			dispatch(addNotification({
				type: "info",
				key: "METADATA_SAVED",
				duration: 3,
				parameter: undefined,
				context: NOTIFICATION_CONTEXT,
			}));
		} catch {
			dispatch(addNotification({
				type: "warning",
				key: "METADATA_NOT_SAVED",
				duration: 3,
				parameter: undefined,
				context: NOTIFICATION_CONTEXT,
			}));
		}
	};
```

But I'd we willing to let that slide as a matter of taste for teaching me about the necessity of `unwrap` here. :D

**However**, if you don't `await`, you don't need `async`, either.